File: level09/Resources/inverse_ascii.go
package main

import (
	"fmt"
	"os"
)

func main() {
	if len(os.Args) != 2 {
		fmt.Printf("Give just one token")
		os.Exit(1)
	}

	for i, c := range []byte(os.Args[1]) {
		fmt.Printf("%c",int(c) + i)
	}
}

File: level09/Resources/inverse_ascii.c
#include <stdio.h>
#include <string.h>

int main(int argc, char **argv)
{
  if (argc != 2)
    printf("Give just one token\n");
  else
  {
    for (int i = 0; i < strlen(argv[1]); i++)
      printf("%c", (int)(argv[1][i]) - i);
    printf("\n");
  }
}
